`moduleResolution`が`Node16`のときimportに失敗する

```sh
$npx tsc
node_modules/maginai/lib/modules/event/cancelable-mod-event.d.ts:16:26 - error TS2834: Relative import paths need explicit file extensions in EcmaScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Consider adding an extension to the import path.

16 import { ModEvent } from './mod-event';
                            ~~~~~~~~~~~~~


Found 1 error in node_modules/maginai/lib/modules/event/cancelable-mod-event.d.ts:16
```
`module: ESNext` && `moduleResolution: Bundler`を使ってもらうことを想定しているのでバグとは言い切れないが
そもそも拡張子ありとなしのimportが混在しているのも微妙という問題がある

--- a/js/mod/modules/event/cancelable-mod-event.js
+++ b/js/mod/modules/event/cancelable-mod-event.js
@@ -1,4 +1,4 @@
-import { ModEvent } from './mod-event';
+import { ModEvent } from './mod-event.js';
 import logging from 'loglevel';
 
 const logger = logging.getLogger('maginai.event');

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,27 +1,22 @@
 {
-  // プロジェクトに合わせて変更してください
   "include": [
     "js/mod/loader.js",
+    "js/mod/modules",
     "types"
   ],
   "compilerOptions": {
-    "target": "ES2020",
-    "module": "ES2015",
+    "target": "ESNext",
+    "module": "Node16",
+    "moduleResolution": "Node16",
     "lib": [
       "ES2020",
       "DOM",
       "DOM.Iterable"
     ],
-    // JSファイルは通常、ソースファイルとして無視されますが、
-    // ここではJSファイルを読み込むようにTypeScriptに指示します
     "allowJs": true,
     // "checkJs": true,
-    // d.tsファイルを生成します
     "declaration": true,
-    // コンパイラを実行すると
-    // d.tsファイルのみ出力されます
     "emitDeclarationOnly": true,
-    "allowUmdGlobalAccess": true,
     "outDir": "./lib"
     // "strict": true,
   }

--- a/tsconfig.typedoc.json
+++ b/tsconfig.typedoc.json
@@ -1,28 +1,21 @@
 {
-  // プロジェクトに合わせて変更してください
   "include": [
-    "./js",
+    "js/mod/loader.js",
+    "js/mod/modules",
     "types"
   ],
   "compilerOptions": {
-    "target": "ES2020",
-    "module": "ES2015",
+    "target": "ESNext",
+    "module": "Node16",
+    "moduleResolution": "Node16",
     "lib": [
       "ES2020",
       "DOM",
       "DOM.Iterable"
     ],
-    // JSファイルは通常、ソースファイルとして無視されますが、
-    // ここではJSファイルを読み込むようにTypeScriptに指示します
     "allowJs": true,
-    // "checkJs": true,
-    // d.tsファイルを生成します
     "declaration": true,
-    // コンパイラを実行すると
-    // d.tsファイルのみ出力されます
     "emitDeclarationOnly": true,
-    "allowUmdGlobalAccess": true,
     "outDir": "./dist"
-    // "strict": true,
   }
 }

